frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.html

```diff
@@ -86,6 +86,34 @@ <h2 class="heading3 push-prev">Write Policy</h2>
                 <wizard-controls></wizard-controls>
             </div>
         </section>
+
+        <section class="column greedy">
+            <div class="column greedy pad">
+                <h2 class="heading3">Caching Configuration</h2>
+                <p class="push-next">
+                    Caching configuration controls how data is cached in local storage
+                </p>
+                <editor class="greedy" params="label: 'Cache TTL'">
+                    <input type="number" min="0" ko.attr.max=maxCacheTTL
+                        class="push-next-half"
+                        ko.value="$form.cacheTTL"
+                        ko.validationCss="$form.cacheTTL"
+                        ko.hasFocus="true"
+                    />
+                </editor>
+                <validation-message params="field: $form.cacheTTL"></validation-message>
+            </div>
+            <div class="row content-middle pad content-box">
+                <p class="remark greedy push-next">
+                    <svg-icon class="icon-small valign-bottom"
+                        params="name: 'notif-info'"
+                    ></svg-icon>
+                    This cache TTL value cannot be changed after the creation of the bucket.
```

Comment:
Update scenario is not included in this PR. Haven't figured out how to implement update in the frontend UI. Should we support enabling/disabling caching besides supporting ttl update?

Suggest support update scenarios in separate PRs.
- changes in API and bucket_server
- Frontend UI changes

Reply:
Yes I agree, let's keep it off this PR.
Good questions. I would think that updating the TTL should be supported, but we don't want to allow switching caching on/off for now. Certainly not from the UI. Maybe we can leave it open to the API so that we can experiment with.

Reply:
Resolving.